admin/tools/docker/configured/scripts/start.sh

```diff
@@ -71,4 +72,11 @@ fi
 # TODO: wmgr.secret should be retrieved at container execution time
 cp $QSERV_RUN_DIR/etc/wmgr.secret.example $QSERV_RUN_DIR/etc/wmgr.secret
 
+# Wait for configure script to run.
+while [ ! -f "$QSERV_RUN_DIR"/configure_done ]
+do
+    echo "start.sh - wait for configure to finish on $(hostname)"
+    sleep 1
+done
```

Comment:
I'm not sure why do you have to loop and wait here, the code above runs in foreground?

Reply:
Once it starts the container, it moves on to the next line. It doesn't wait for the container, so wait.sh would try to run qserv-status.sh before the container is done with qserv-configure.py, which is really confusing and probably not safe either. Waiting for the qserv-configure.py to finish forces things to run in a consistent order.